Backend/src/main/java/idealab/api/repositories/PrintJobRepo.java

```diff
@@ -15,5 +16,10 @@
     List<PrintJob> findPrintJobByStatus(Status s);
     List<PrintJob> findAll();
     List<PrintJob> findByCustomerInfo(CustomerInfo customerInfoId);
-    
+
+
+    @Query(value = "SELECT p.* FROM print_job p " +
+                    "where p.fk_customer_info_id = ?1 " +
+                    "order by p.created_at DESC", nativeQuery = true)
+    List<PrintJob> findPrintJobsByCustomerIdNewestFirst(Integer id);
```

Comment:
Everything looks great! My only question is could we add a test for this to ensure it isn't throwing an error? I think it would be good to do for any that we are manually querying in case someone changes the constructor or table without realizing the repercussions

Reply:
I think what you're looking to do falls into the category of functional/integration testing vs unit testing.